### DISCUSSION TITLE
List of all available pre-trained model weights in one "model catalog" location

### DISCUSSION THREAD

**Discussion by stephenleo** [Lightning-AI/pytorch-lightning#11477] [General]
Hi All, I'm coming from an NLP background where I routinely use [Huggingface models](https://huggingface.co/models) to discover new interesting pre-trained models. Recently I've been trying to use the pre-trained SimCLR weights from `pl_bolts`, and though the documentation is quite spot on [Link](https://pytorch-lightning-bolts.readthedocs.io/en/latest/self_supervised_models.html), it got me wondering if there is a convenient catalog of all pre-trained models available here? IMHO, I think it will help tremendously with model discovery. Thanks!

**Comment by akihironitta** [comment]
Hi @stephenleo, unfortunately, all models defined in Lightning Bolts have been deprecated in https://github.com/PyTorchLightning/lightning-bolts/issues/719.

**Reply by stephenleo** [reply]
Ah I see. It makes sense. I didn't notice any alternative mentioned. May I check if there is any alternative in plan outside of lightning bolts?
